ko/admin/config.rst

```diff
@@ -2375,6 +2379,23 @@ HA 관련 파라미터
 
  **SHOW CREATE TABLE** 구문과 같이 인덱스 구문 정보를 보여줄 때 **WITH** 절의 옵션 정보의 출력 여부를 지정한다. 기본값은 NO이다. 단, unloaddb 툴에서는 이 설정값의 영향을 받지 않는다.
 
+**enable_memory_monitoring**
+
+ **enable_memory_monitoring**\은 서버의 힙 메모리 사용량 모니터링 여부를 지정한다. 이 값을 YES로 설정하면, 서버의 메모리 모니터링 기능이 활성화되어 서버의 힙 메모리 사용량을 지속적으로 추적 관리한다. 힙 메모리 사용량은 큐브리드 소스 코드에서 메모리 동적 할당이 발생하는 파일과 라인을 기준으로 추적되며, 동일한 위치에서 여러 번 메모리 할당이 발생하는 경우 메모리 사용량은 누적된다. 추적 중인 메모리가 해제되면 누적된 총 메모리 사용량에서 해제된 메모리 크기 만큼 감산하여 실시간 힙 메모리 사용량을 계속 추적한다. 추적 중인 힙 메모리 사용량은 :ref:`memmon` 유틸리티를 통해 확인할 수 있다. 기본값은 NO이다.
+
+.. note::
+
+    *   Linux 계열에서만 사용할 수 있다.
+    *   메모리 사용량을 추적하기 위해 매크로를 이용한 코드 수준의 자동화 과정에서 glibc와의 충돌이 발생하였다. 이를 막기 위해 glibc 내부에서 발생하는 메모리 사용량(STL 컨테이너)은 추적하지 않으며, 동일한 이유로 헤더 파일에서 발생하는 메모리 사용량은 추적하지 않는다.
```

Comment:
How about removing this content from the manual, since this content is briefly explained in the following notice ?

Reply:
@hornetmj I think it needs discussion.

Reply:
@mhoh3963 Is it okay if we don't explicitly specify the constraints? Or, how about combining the two NOTES into one and removing the redundant parts?

Reply:
I think there is no need to specify detailed restrictions, because the below notice is already described  "큐브리드의 메모리 사용량 모니터링 기능이 헤더 파일 및 glibc 내부에서 발생하는 메모리 할당을 추적하지 않기 때문이며".

Reply:
@mhoh3963 What I would like to ask is whether it is also necessary to remove the following two constraints.
- Linux 계열에서만 사용할 수 있다.
- HA 환경 구성 시 슬레이브 노드는 마스터 노드의 로그를 처리하는 과정에서 메모리 할당 비중이 높다. 이로 인해 메모리 사용량 모니터링의 비용이 증가하여 성능 문제가 발생할 수 있다. 따라서 슬레이브 노드에서는 사용을 권장하지 않는다.

If that is not the case, I was wondering if it would be possible to consolidate the two separate 'Notes' into a single 'Note'.

Reply:
@mhoh3963 @hornetmj @joohok I revise the note. Please check it.